contribute.md

```diff
@@ -134,7 +149,7 @@ We're trying to maintain high quality, authoritative documentation in the Forema
 Both the manual and the web site are contained in our [theforeman.org repository](https://github.com/theforeman/theforeman.org).  See the README.md to get started.  Contributions to this project are licensed under Creative Commons Attribution-ShareAlike 3.0.
 
 ### Design and User interface
-Foreman has a very basic design and user interface. Any improvements to it including new themes, skinning, or interface adjustments could help every user.
+Foreman has a very basic design and user interface. Any improvements to it including new themes, skinning, or interface adjustments could help every user. Design discussion should happen on [the dev mailing list](/support.html#Mailinglists).
```

Comment:
Nitpick, the first sentence has been on these docs since the early days when Foreman was all 'blue' (dunno how to call that nor I do remember the version, 0.9?), I don't think it holds true anymore? 
I'd also mention we are trying to follow Patternfly guidelines (http://patternfly.org/) so any help with that would be appreciated